client/components/layouts/DocumentSidebar.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+function DocumentSidebar({ access }) {
+  const homePage = location.pathname === '/home' ? ' active' : '';
+  const newDocumentPage = location.pathname === '/document/new'
+    ? ' active' : '';
```

Comment:
You may need a utility function like:
`function makeActive (path) {
    return location.pathname === path ? ' active' : '';
}`